src/main/java/com/zzpj/backend/entities/Purchase.java

```diff
@@ -0,0 +1,43 @@
+package com.zzpj.backend.entities;
+
+import javax.persistence.*;
```

Comment:
We should specify imports:
`import javax.persistence.Column;`

Reply:
intelij automatically generate these import even though i put one by one